pages/linux/brew.md

```diff
@@ -10,13 +10,13 @@
 
 `brew install {{formula}}`
 
-- List installed formulae [with matching name]:
+- List installed formulae, argument `formula` is optional:
 
-`brew list {{[text]}}`
+`brew list {{formula}}`
```

Comment:
Or you can follow the structure of the below example and say "(passing no formula lists all installed formulae)" at the end. I think that's okay too.